server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java

```diff
@@ -504,12 +504,15 @@ static long loadFiles(Loader loader, BulkInfo bulkInfo, Path bulkDir,
     }
 
     long sleepTime = loader.finish();
-    if (sleepTime > 0) {
-      log.trace("{}: Tablet Max Sleep is {}", fmtTid, sleepTime);
-      long scanTime = Math.min(totalProcessingTime.toMillis(), 30_000);
-      log.trace("{}: Scan time is {}", fmtTid, scanTime);
-      sleepTime = Math.max(sleepTime, scanTime * 2);
+    if (sleepTime <= 1) {
+      log.trace("{}: Sleeping for {}ms", fmtTid, sleepTime);
+      return sleepTime;
     }
+
```

Comment:
The content of this block is duplicated below. I think the equivalent change would have kept the previous if statement, and just changed it to the following, rather than duplicate the last two lines.

```java
    if (sleepTime > 1) {
```

Reply:
Yeah, it looks like `0` or `1` are really success cases in the OnlineLoader and you don't really want to sleep.